### ISSUE TITLE
Fix wcag for Appbar.

### ISSUE THREAD

**Issue by kaminderpal** [Canadian-Geospatial-Platform/geoview#2402]
When tabbing on appbar and panel focus goes out of panel, should be focus trapped inside.
